js/mainjs/trayMenu.js

```diff
@@ -8,15 +8,22 @@ module.exports = function (window) {
 	var menutemplate = [
 		{
 			label: 'Show Sia',
-			click: function () { window.show(); }
+			click: function () { window.restore(); }
 		},
 		{ type: 'separator' },
 		{
 			label: 'Hide Sia',
-			click: function () { window.hide(); }
+			click: function () { window.minimize(); }
 		},
+		{ type: 'separator' },
+		{
+			label: 'Quit Sia',
+			click: function() {
+				window.wantsQuit = true;
+				window.close();
```

Comment:
What about creating a custom event to handle quitting? Something like `window.quit`.